neddf/scripts/dataset_visualizer.py

```diff
@@ -0,0 +1,397 @@
+import os
+from pathlib import Path
+from typing import Final, List
+
+import hydra
+import numpy as np
+import open3d as o3d
```

Comment:
when I run `poetry run python3 neddf/scripts/dataset_visualizer.py` at my PC, following error is raised. I guess we should add `open3d` to `pyproject.toml` and `poetry.lock` file.
```
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.11) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Traceback (most recent call last):
  File "neddf/scripts/dataset_visualizer.py", line 7, in <module>
    import open3d as o3d
ModuleNotFoundError: No module named 'open3d'
```

Reply:
I updated poetry.lock, please confirm it works!

Reply:
Thank you for your update!